packages/flutter_adaptive_scaffold/CHANGELOG.md

```diff
@@ -6,9 +6,8 @@
 * Add `navigationRailDestinationBuilder` to apply custom Destinations.
 * Add `groupAlignment` property to change alignment.
 * Set `navRailTheme` when using the Drawer just like the other NavigationRails.
-
-## 0.1.11+1
```

Comment:
Actually, you need to leave this version here: it is the current published version, and it needs to represent what was actually published.